### DISCUSSION TITLE
Controls not displaying after V11.0 upgrade.

### DISCUSSION THREAD

**Discussion by Ryan-K-94** [AvaloniaUI/Avalonia#12814] [Q&A]
I have upgraded a project I'm working on to V11.0.4. Have made all necessary changes as per the upgrade guide to get it building again. 

For some reason some of the controls aren't working/displaying when I run the application. The main ones I have come across so far are Textbox, MaskedTextBox and ComboBox. 

TextBlock, ListBox, DataGrid, Button, Menu TabControl are all working fine. 

Have made a new V11.0 project and tested the controls in that and they are all working fine. So it's something specific to the upgraded project. Have burnt a couple of days on this now, any advice would be greatly appreciated.

**Comment by timunie** [comment]
Any custom stlyes included? Either somewhere in your App or in a 3rd party lib?

**Reply by Ryan-K-94** [reply]
Looks like it's this that I had defined in App.axaml

		<Style Selector="DataValidationErrors">
			<Setter Property="Template" Value="{DynamicResource TooltipDataValidationContentTemplate}" />
			<Setter Property="ErrorTemplate" Value="{DynamicResource TooltipDataValidationErrorTemplate}" />
		</Style>

How's best to approach this in V11.0?

**Reply by timunie** [reply]
lookup original ValidationError style and update template accordingly
